### ISSUE TITLE
Add an option to remove the "Reply contact is.." notification

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#260]
```
===  IMPORTANT - READ THIS!  ===
Please use the search feature to confirm that your request isn't entered
already. You can cut this part of your report.
================================

Please describe your idea here:

I would like an option to be able to disable the "Reply contact is ..." 
notification that occurs when you receive a text message from a different 
sender from your current reply contact.

I never use the Reply command so it is simply annoying to have the window 
pop-up telling me something that I don't care about.

Great application, by the way :)

```

Original issue reported on code.google.com by `jagriff...@gmail.com` on 31 Mar 2012 at 9:54
- Blocked on: #240

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 31 Mar 2012 at 10:03
- Changed title: **Add an option to remove the "Reply contact is.." notification**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Already fixed in the version 4.4.3.
You have to disable the recipient command by executing "deactivate:recipient" 
or in the command tab.

When the command is disabled, no notification is sent
```

Original comment by `Florent....@gmail.com` on 10 Nov 2013 at 4:34
- Changed state: **Deployed**
- Now blocked on: #240
